### ISSUE TITLE
[FastPitch1.1/Pytorch]  Document of the preprocessing and training from scratch

### ISSUE THREAD

**Issue by JohnHerry** [NVIDIA/DeepLearningExamples#1140]
Related to **Model/Framework(s) or something else (describe)** 

*Examples:* 
* *FastPitch1.1/PyTorch* 
* *AMP* 
* *Pytorch1.10.2+cu111*
* *Jupyter notebooks*

**Is your feature request related to a problem? Please describe.**
A clear a
nd concise description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**

**Describe alternatives you've considered**

I want to know is there any document to say the  max and min size of audio samples?  It seems there be. because I had been broken the training process for many many times. for same sample audios may be less then 1 second or  more then 17 seconds. I do not clearly know the limitations .  I had used the  `prepare_dataset.py` to get pitch and mel  before my training run. But this script did not filtered out the too short or too long audios at all.

So, Is there any such gates? what are they?

**Additional context**

When my training broken, It always stop at the code "fastspeech/model.py    assert torch.all(torch.eq(dur_tgt.sum(dim=1),mel_lens))"
for the last value of the batch, the mel len and  dur_tgt.sum is not equal. Then I find the sample file about this, It will be too long or too short.

**Comment by JohnHerry** [comment]
Document Error:
`
--energy-conditioning - enables additional conditioning on energy (default: enabled)
`
No!!!  The default value of --energy-conditioning when training is  FALSE!!!
